greenpithumb/humidity_sensor.py

```diff
@@ -0,0 +1,27 @@
+import dht11_exceptions
+
+class HumiditySensor(object):
+	"""Wrapper for humidity sensor."""
+
+	def __init__(self, dht11_result):
+		"""Creates a new HumiditySensor wrapper.
+
+		Args:
+			dht11_result: Result of a reading from a DHT11 humidity and
+			temperature sensor.   
+		"""
+		self._dht11_result = dht11_result
+
+	def get_humidity_level(self):
+		"""Returns humidity level."""
+
+		if self._dht11_result.error_code() == 1:
```

Comment:
Once we import the DHT11 library, we'll want to replace these with the constants the library exports. Can we add a comment like

```
# TODO(JeetShetty): Replace these with constants from dht11 module
```

For fun, you can also create an [issue](https://github.com/JeetShetty/GreenPiThumb/issues) if you want to see what that's like.